cpp/tests/binaryop/binop-compiled-test.cpp

```diff
@@ -26,9 +26,9 @@
 #include <cudf_test/column_wrapper.hpp>
 #include <cudf_test/type_lists.hpp>
 
+#include "cudf/utilities/error.hpp"
```

Comment:
Should this not have been the following?
```suggestion
#include <cudf/utilities/error.hpp>
```

Reply:
Actually, no. This can't be committed without also moving this to line#23.